### ISSUE TITLE
Setting: NPC Soul Shuffle

### ISSUE THREAD

**Issue by Revvenn** [OoTMM/OoTMM#548]
Just using this as a means to track NPC Soul Shuffle.

- [x] Initial implementation for OOT with most important NPC Souls
- [x] Make Zelda and Sheik Souls required for Light Arrow Cutscene check
- [x] Add the child carpenters in Kakariko Village to the current Carpenters Soul
- [x] Add Grog (OOT) Soul
- [x] Add the rest of the civilians to Hylian Civilian soul (kid chasing cucco in Market, Honey & Darling in market, the crowds in Market, etc)
- [x] Add the Lake Hylia Lab guy Soul (do not know his name at all lol it's the guy you give Eyeball Frog to)
- [x] Make Zelda's Soul be required for the Song of Time and Ocarina of Time checks
- [x] Add Kak Potion Shop Witch Soul
- [x] Add Bean Salesman Soul
- [x] Add Bombchu Merchant Soul
- [x] Add Ingo Soul
- [x] Initial implementation for MM with most important NPC Souls (most likely basically every sidequest NPC, Clock Town NPCs, and the various regional based ones such as Goron, Zora, Scrubs, and mini-games)
- [x] Add more MM NPC Souls (whatever may be missing from the initial implementation above, will adjust/edit this based on that)
- [x] Add Shared NPC Souls (this would likely be applicable to mostly things like Anju, Talon, Malon, Gorman/Ingo, Bowling Lady with Treasure Chest Game Lady in MM, NPCs that are obviously lookalikes and could be considered or **are** the same person, etc)
- [x] Add Gerudo Soul
- [ ] Add Scarecrow Soul
- [ ] #832

**Comment by Nax** [comment]
I think Celeste worked on that a bit, is the list up-to-date?

**Comment by Revvenn** [comment]
> I think Celeste worked on that a bit, is the list up-to-date?

She's got a list of souls together and I suggested several more to her -- this was the base list but since she did take up doing some of this, I was able to get more specific with her. However, she only has a list together in the code and hasn't actually started anything (but obviously she is going to start soon-ish).

**Comment by CelestialKitsune** [comment]
I am on the fence for the last few souls, I'll be honest. Not really sure if they're worth the hassle.

**Comment by Revvenn** [comment]
> I am on the fence for the last few souls, I'll be honest. Not really sure if they're worth the hassle.

Yeah, really **honestly**, the only glaring one on the list is the Gerudo/Pirates considering they _are_ a race like Goron/Zora/Scrubs/etc. And I understand there's technical issues with those because of the OOT patrollers vs the OOT and MM fighters vs MM the boat riders vs OOT leader as well as the beehive room in particular. All the others I don't mind not having. Cows included lol. I figure maybe if Gerudo/Pirates are enough of a hassle on their own, they should just be the last one imo.

**Comment by CelestialKitsune** [comment]
Scarecrow is not getting done. Converted the last bit into its own issue.
